protocol/x/rewards/keeper/keeper.go

```diff
@@ -126,26 +128,45 @@ func (k Keeper) GetRewardShare(
 //     by quote quantums of the fill.
 func (k Keeper) AddRewardSharesForFill(
 	ctx sdk.Context,
-	takerAddress string,
-	makerAddress string,
-	bigFillQuoteQuantums *big.Int,
-	bigTakerFeeQuoteQuantums *big.Int,
-	bigMakerFeeQuoteQuantums *big.Int,
+	fill clobtypes.FillForProcess,
+	revshares []revsharetypes.RevShare,
 ) {
 	// Process reward weight for taker.
 	lowestMakerFee := k.feeTiersKeeper.GetLowestMakerFee(ctx)
 	maxMakerRebatePpm := lib.Min(int32(0), lowestMakerFee)
+
+	totalNetFeeRevSharePpm := uint32(0)
+	affiliateRevShareQuoteQuantums := big.NewInt(0)
+	for _, revshare := range revshares {
+		if revshare.RevShareFeeSource == revsharetypes.REV_SHARE_FEE_SOURCE_NET_FEE {
+			totalNetFeeRevSharePpm += revshare.RevSharePpm
+		}
+		if revshare.RevShareType == revsharetypes.REV_SHARE_TYPE_AFFILIATE {
+			affiliateRevShareQuoteQuantums = revshare.QuoteQuantums
+		}
+	}
+	remainingRevSharePpm := lib.OneMillion - totalNetFeeRevSharePpm
```

Comment:
Variable naming doesn't make sense. We aren't computing remaining revenue shares
```suggestion
	remainingRevSharePpm := lib.OneMillion - totalNetFeeRevSharePpm
```

Reply:
Terrible naming on my part